query/sql/sql_instance_postgresql_log_min_messages_database_flag_set_minimum.sql

```diff
@@ -0,0 +1,21 @@
+select 
```

Comment:
Change query name to `sql_instance_postgresql_log_min_messages_database_flag_error.sql`